include/ck/tensor_operation/gpu/grid/gridwise_batched_gemm_softmax_gemm_xdl_cshuffle_v1.hpp

```diff
@@ -717,6 +717,11 @@ struct GridwiseBatchedGemmSoftmaxGemm_Xdl_CShuffle
                                                                    blockwise_gemm,
                                                                    acc_thread_buf,
                                                                    num_k_block_main_loop);
+
+            // Acc0 elementwise Op
+            static_for<0, acc_thread_buf.Size(), 1>{}(
+                [&](auto i) { acc_element_op(acc_thread_buf(i), acc_thread_buf[i]); });
```

Comment:
The pointwise scaling is already performed by a1_blockwise_copy one K-slice at a time. Is there a need to move it outside and scale the whole tile at once?

Reply:
in previous implementation, Acc0ElementOp is actually performance on the output of softmax. This PR apply Acc0ElementOp to the output of GEMM0

Reply:
I made a mistake. Thanks for pointing out the difference.